gqcp/include/Basis/Transformations/GTransformation.hpp

```diff
@@ -186,4 +187,19 @@ struct JacobiRotatableTraits<GTransformation<Scalar>> {
 };
 
 
+/*
+ *  MARK: OrbitalRotationGeneratorTraits
+ */
+
+/**
+ *  A type that provides compile-time information related to the abstract interface `OrbitalRotationGenerators`.
```

Comment:
```suggestion
 *  A type that provides compile-time information on orbital rotation generators that is otherwise not accessible through a public class alias.
```

`OrbitalRotationGenerators` is not an abstract class interface.